### PR TITLE
Fix some clippy warnings in `components/script/webdriver_handlers.rs`

### PR DESCRIPTION
**Pull request by komutumu**
Fixed some clippy warnings in components/script/webdriver_handlers.rs

---
- [X] `./mach build -d` does not report any errors
- [X] `./mach test-tidy` does not report any errors
- [X] These changes are part of #31500 
- [X] These changes do not require tests because they do not touch functionality, they only fix lint errors.

### CODE REVIEW

**Comment by jahielkomu** [comment]
Hello @mrobinson . I hope you are well.
Please kindly review.
Thank you

File: components/script/webdriver_handlers.rs [hunk 1]
```diff
@@ -382,10 +382,7 @@ fn get_element_in_view_center_point(element: &Element) -> Option<Point2D<i64>> {
         .map(DomRoot::upcast::<Element>)
         .and_then(|body| {
             element
-                .GetClientRects()
-                .iter()
-                // Step 1
```
Comment (sagudev) [inline components/script/webdriver_handlers.rs:387]: This comment refers to first step in spec:
> Let rectangle be the first [element](https://dom.spec.whatwg.org/#concept-element) of the [DOMRect](https://www.w3.org/TR/geometry-1/#domrect) sequence returned by calling [getClientRects](https://www.w3.org/TR/cssom-view-1/#dom-element-getclientrects)() on [element](https://dom.spec.whatwg.org/#concept-element).

so it should be kept.

Comment (jahielkomu) [reply components/script/webdriver_handlers.rs:387]: Thank you @sagudev .
Your response helps a lot.

I'm working on it now.

**Comment by jahielkomu** [comment]
Hello @mrobinson and @sagudev 
Please kindly review.
Thanks.

File: components/script/webdriver_handlers.rs [hunk 2]
```diff
@@ -381,12 +381,9 @@ fn get_element_in_view_center_point(element: &Element) -> Option<Point2D<i64>> {
...
         .and_then(|body| {
-            element
-                .GetClientRects()
-                .iter()
-                // Step 1
-                .next()
-                .map(|rectangle| {
+           let rectangle_element = element.GetClientRects();
+           let rectangle_option = rectangle_element.iter().next();
+           rectangle_option.map(|rectangle| {
```
Comment (mrobinson) [inline components/script/webdriver_handlers.rs:386]: I believe that @sagudev meant that you should keep the comment. I recommend something like you had before, just restoring the original comment. We could even improve it a bit:

```
        // Step 1: Let rectangle be the first element of the DOMRect sequence 
        // returned by calling getClientRects() on element.
        element
            .GetClientRects().first()
            .map(|rectangle| {
```

**Review by mrobinson — CHANGES_REQUESTED** [review]

**Comment by jahielkomu** [comment]
@mrobinson in the comment; 
// Step 1: Let rectangle be the first element of the DOMRect sequence returned by calling getClientRects() on element, should ```element```, ```DOMRect``` and ```getClientRects()``` be links as earlier portrayed by @sagudev ?

**Comment by mrobinson** [comment]
> I have updated the PR @mrobinson, please review.
> 
> @mrobinson in the comment; // Step 1: Let rectangle be the first element of the DOMRect sequence returned by calling getClientRects() on element, should `element`, `DOMRect` and `getClientRects()` be links as earlier portrayed by @sagudev ?

This is not necessary. Normal comments aren't processed into rustdoc, so it isn't always useful to make links. In addition, the link exist in the specification that is linked to. I think these links are in @sagudev's comment because they were preserved while copying and pasting from the specification.

**Review by mrobinson — APPROVED** [review]

**Review by sagudev — APPROVED** [review]

**Pull request merged**